### DISCUSSION TITLE
Background Picture, Entity Icons and Icons on Main/Home Page

### DISCUSSION THREAD

**Discussion by T3mplate-wow** [Blackymas/NSPanel_HA_Blueprint#368] [General]
Hey, I didn’t find any documentation for this, so I am asking here:
* On the readme page I find the text: „Modern design - design easy to change via Adobe Express (free + design template)“ -> Where can I change this? Which resolution or format has to be used?
* Also on the readme page: „Top menu with 10 icons for specific states“ -> Where can I configure this?
* The last one is maybe a feature request. For the new entity page the Icons can be changed. Is this also possible for the Button Pages? I used there input_buttons for different scenes and it would be great to have there button icons for my scenes / automations behind this.

**Comment by T3mplate-wow** [comment]
@ @Blackymas is there Doku existing somewhere?

**Comment by Blackymas** [comment]
will only be available with version 3.2.

**Comment by JohannesHoffmann** [comment]
Hi, i stumbled accross the same question. Where to find an documentation on how to change the design with Adobe Express?
Would love to help creating a documentation page if there are some resources i can read?
